docs/_renderer.py

```diff
@@ -184,6 +183,15 @@ def signature(self, el: dc.Function, source: Optional[dc.Alias] = None):
         return super().signature(el, source)
 
 
+def render_definition_list(rows: list[tuple[str, str]]) -> str:
+    def indent_newlines(s: str) -> str:
+        return s.replace("\n", "\n    ")
+
+    out = [f"{param}\n\n:   {indent_newlines(desc)}" for param, desc in rows]
+
+    return "\n\n".join(out)
+
+
```

Comment:
quartodoc now (v0.7.2) has a [pandoc subpackage](https://github.com/machow/quartodoc/tree/678177779eaa5a02fdd74de42c7fe88cdfbf7884/quartodoc/pandoc) that you can use to generate markup.

For definition lists check out examples in the [test cases](https://github.com/machow/quartodoc/blob/678177779eaa5a02fdd74de42c7fe88cdfbf7884/quartodoc/tests/pandoc/test_blocks.py#L192-L285)

Reply:
Thanks @has2k1! The pandoc subpackage looks awesome and I'm excited to use it. Unfortunately when I upgraded py-shiny to quartodoc 0.7.2 and griffe to 0.33, I get the following error:

```
AttributeError: module 'griffe.expressions' has no attribute 'Expression'. Did you mean: 'get_expression'?

#...

  File "/Users/garrick/work/posit-dev/py-shiny/venv/lib/python3.11/site-packages/quartodoc/renderers/md_renderer.py", line 150, in signature
    res = self.signature(el.obj)
          ^^^^^^^^^^^^^^^^^^^^^^
# ...

  File "/Users/garrick/work/posit-dev/py-shiny/docs/_renderer.py", line 185, in signature
    return super().signature(el, source)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#...

beartype.roar.BeartypePep563Exception: Stringified type hint 'exp.Expression' syntactically invalid (i.e., AttributeError("module 'griffe.expressions' has no attribute 'Expression'")).
```

I'm hoping those are the most relevant parts of the stack trace, and maybe you or @machow have an idea what's goin on.

The most relevant line in our code is here: https://github.com/posit-dev/py-shiny/blob/88deca4a52dda23953ef12d181a121d1ef0aa842/docs/_renderer.py#L183-L184

Reply:
griffe 0.33 introduced API changes to the expressions module.

You need to

1. Change these annotations

```
expr.Expression to expr.Expr
expr.Name to expr.ExprName
```
2. Change the attributes of `expr.ExprName` objects

```
el.source to el.name
el.full to el.canonical_path
```

The equivalent change quartodoc is https://github.com/machow/quartodoc/pull/304/files

Reply:
That worked perfectly, thanks @has2k1!